src/main/java/com/pmm/budgetcalculator/expensing/repositories/Expense.java

```diff
@@ -1,15 +1,17 @@
-package com.pmm.budgetcalculator.expensing.domain.expense;
+package com.pmm.budgetcalculator.expensing.repositories;
 
-import com.pmm.budgetcalculator.foundation.BaseEntity;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 import org.springframework.format.annotation.DateTimeFormat;
 
-import javax.persistence.Entity;
+import javax.persistence.*;
 import java.math.BigDecimal;
 import java.util.Date;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
```

Comment:
This is confusing - imports without use. To avoid this:              1. Go to settings -> editor -> auto import and select optimize imports on the fly.                                                    or                                                                   2. settings -> version control -> commit  -> before commit, check Optimize imports